keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -729,6 +736,87 @@ that might indicate a serious problem?
 
 #### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+We have unit tests testing the "rollback" scerario as we do test with positive and negative cases of the feature.
```

Comment:
```suggestion
We have unit tests testing the "rollback" scenario as we do test with positive and negative cases of the feature.
```

Reply:
Gasp.  Why wouldn't verify-spelling not catch that!?

Reply:
updated.  Please re add lgtm if you get a chance.